schema/divisions/defs.yaml

```diff
@@ -89,6 +89,26 @@ description: Common schema definitions for divisions theme
           type: string
           minLength: 1
           pattern: ^(\S.*)?\S$    # Leading and trailing whitespace are not allowed.
+    perspectives:
```

Comment:
Why move this property into `defs.yaml`. It isn't expected to be used anywhere else, which is why it was modeled directly into `division`.

Reply:
It is right above `perspective` I wanted to keep them coupled as I don't see a point in using one without the other.

Reply:
Seems unrelated to the main purpose of the PR.

Reply:
It will be used in boundaries since they will also depend on the perspective of the given country. I will add support for that in the next PR.